crates/bevy_remote/src/schemas/mod.rs

```diff
@@ -1,4 +1,68 @@
 //! Module with schemas used for various BRP endpoints
 
+use bevy_asset::{ReflectAsset, ReflectHandle};
+use bevy_ecs::{
+    reflect::{ReflectComponent, ReflectResource},
+    resource::Resource,
+};
+use bevy_platform::collections::HashMap;
+use bevy_reflect::{
+    prelude::ReflectDefault, Reflect, ReflectDeserialize, ReflectSerialize, TypeData,
+    TypeRegistration,
+};
+use core::any::TypeId;
+
 pub mod json_schema;
 pub mod open_rpc;
+
+/// Holds mapping of reflect data types to strings,
```

Comment:
I think we should use the term "type data" instead. Data types can be confused with actual [data types](https://doc.rust-lang.org/book/ch03-02-data-types.html).

In documentation we should also be sure to link "type data" back to `TypeData`.
```suggestion
/// Holds mapping of reflect [type data](TypeData) to strings,
```